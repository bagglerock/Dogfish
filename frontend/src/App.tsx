import React, { useEffect } from 'react';
import axios from 'axios';
import './App.scss';

export const App: React.FC = () => {
  useEffect(() => {
    axios.get('/api/test').then(response => {
      console.log(response);
    });
  }, []);

  return (
    <div className="app">
      <p>Hangman, Coming soon...</p>
    </div>
  );
};
