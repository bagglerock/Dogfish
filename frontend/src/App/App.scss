html,
body {
  height: 100%;
}

body {
  background-color: rgb(1, 20, 1);
  background-image: url('/images/oldcomputer.gif');
  background-repeat: repeat;
}

h1,
h2,
h3,
h4,
h5 {
  color: darkolivegreen;
  font-family: 'Press Start 2P', cursive;
}

@import './Layout/Header/header.scss';
@import './Layout/Main/main.scss';
@import './Layout/Footer/footer.scss';
@import './Game/game.scss';
