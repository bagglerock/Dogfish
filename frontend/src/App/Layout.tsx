import React from 'react';

export const Layout: React.FC = () => {
  const date = new Date();
  const year = date.getFullYear();

  return (
    <>
      <header>
        <h1>Hangman</h1>
      </header>

      <main>
        <p>Hangman, coming soon..</p>
      </main>

      <footer>
        <p>&copy; Oscar Villalta {year}</p>
      </footer>
    </>
  );
};
