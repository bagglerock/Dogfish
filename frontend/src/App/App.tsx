import { Layout } from 'app/Layout/Layout';
import React from 'react';
import './app.scss';

export const App: React.FC = () => (
  <div className="app">
    <Layout />
  </div>
);
