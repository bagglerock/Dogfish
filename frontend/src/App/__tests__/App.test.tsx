import { render } from '@testing-library/react';
import { App } from 'app/App';
import React from 'react';

test('renders App/Layout', () => {
  const { container } = render(<App />);

  expect(container).toMatchSnapshot();
});
