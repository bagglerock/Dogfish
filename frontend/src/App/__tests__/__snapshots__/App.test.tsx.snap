// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App/Layout 1`] = `
<div>
  <div
    class="app"
  >
    <header
      class="header"
    >
      <h1>
        Hangman
      </h1>
    </header>
    <main
      class="main"
    >
      <p>
        Hangman, coming soon..
      </p>
    </main>
    <footer
      class="footer pt-3"
    >
      <h1>
        Â© Oscar Villalta 
        2021
      </h1>
    </footer>
  </div>
</div>
`;
